ugo_collector/RatePlan_Updated/RatePlanUpdated_1_firstAnalysis.ipynb

```diff
@@ -0,0 +1,2320 @@
ehkAggKNjzQKPUbFiZURGhiu0h4eHQU1NDRUqVPzi+RJJ1j4fQmHWaoq6detj8+aAHOPV1ISyAkyVKoCrqxt69fLCqlVLUadOXejq6gIAXF1r4ebN67h69TKaNm2uME5a2kfcvHkD1taVeasKERGVKNx4lIiIiFROcPBxXL0agiZNPGFpWbrA4zRo0Ajh4WF48uQfWdunT5/wxx+n4OJSC3p6+rnmIRQKYW9fDQBgaGgEe/uqcn++RFdXD0OHjkBcXBwOHtwva2/btiOMjIyxfv0axMfHK5y3du0qJCUlok+fAfm4WiIiouKPKzmIiIio2MrMlOD+/XsAgPT0NLx58xohIRdx+fJF1KjhhIkTp33V+G3bdsDhwwcwdeokDB06HAYGBjhwYC9iY2MxZ84iWdyZM8EICbmIOnXqoVQpCyQmJuDUqZO4fPkCevbsA0tLywLn0KxZS+zZsxN79+5Aly5e0NXVg4GBAebNW4zJk8fip5/6oEePPqhc2QYJCQk4eTIIISGX4O3dA82atfyq6yciIipuWOQgIiKiYis1NQVDh2atVtDW1oaxsQnEYjvMnr0AjRt7Qij8ukWrWlpaWLlyHdauXYmlSxchPT0NYrE9/PzWwN6+iiyudGkrxMcnYN26VUhISICGhiYqV7bBtGmz0KpV26/KQSgUYsiQ4Zg4cTQOHNiLfv0GAcjaQHX79j3YsWMb9uzZgbi4WGhr68DevgoWLVqGevUaftW8RERExZFA+rUPb1dhcXHJkEj48nxv5uYitGk3s6jTyJfjQbPRuP/cok4jz85v/wUxMUlFnQYRfQNRUc9gaVlBaZ9IpAVtbc3vnJG8jx/TkZSUVqQ5kLwvvWeIiIh+REKhAKamym8Z5UoOIiKiEiIpKY0FBiIiIlJp3HiUiIiIiIiIiFQCixxEREREREREpBJY5CAiIiIiIiIilcAiBxERERERERGpBBY5iIiIiIiIiEglsMhBRERERERERCqBRQ4iIiIiIiIiUgkschARERERERGRSmCRg4iIiIiIiIhUgnpRJ0BERETfh8hAC9pamkWaw8e0dCQlphXo3BMngrBgwWxs27YLtrZ2mD9/Fi5fvoDg4AtycXfv3sHkyWNRurQV/PxWw9jYRNZ3795f8PUdBB0dXfz+ezB0dXWVznXr1nVs2rQeYWFPoKuriwYNGsHXdxREIpEs5s6d2xg1aqjS83ftCkSFChW/eD3z58/CyZPHZMdqamowNTWDk5MzBg70Qdmy5RSu/XNGRsaoXNkWffsOgIuL2xfnIiIiKilY5CAiIiohtLU00bj/3CLN4fz2X5CEghU58uLatRBMnz4ZYrEdlixZKVeUAIDjx48CAFJTU3Du3Bm0bdtBYYw7d25jwoTRqF+/EQYP9kVsbAzWr/dHZGQ41qzZDKFQfiGsr+9IODo6y7VZWpbOU746Orrw81sDAMjMzMCLF8+wdesmjB7ti4CAvdDT05eLnz59NsqWLQ9AitjYGOzbtxvjx4/EunVbUKVKtTzNSUREpMpY5CAiIiKVcPbsGcyd+wucnV2xYMFSaGtry/Wnpqbi3Lk/UL26I54+jcTx40eVFjnWrVsFa+vKmDNnoaygYWZmhrFjR+D8+T/QtGlzufhy5SrAwaF6gXJWUxPKnevoWBMikSGmTZuIe/f+hru7h1x85co2sLW1kx07O7uiTRtPXLhwjkUOIiIicE8OIiIiUgFBQUcwe/Y01K1bH4sX+ykUOADg/Pk/kJLyAR06dIanZwvcu/cXnj9/JhcTE/MWjx49RIsWreVWbLi5ucPcvBQuXDj3za9FT08PAJCR8SnXWG1tHQiFwjzFEhERlQQschAREVGxtmfPTixePA8tW7bBnDmLoKGhoTTu+PGj0NXVQ6NGTdGmTXtZ2+ciIsIBANbWNgrnW1vbIDIyXKH9118XoGHD2mjRoiEmTRqDx48f5Sv/jIwMZGRkIC0tDU+ehGLz5vUwNTWDs7OrQmxmpkQWHxX1Bn5+SwAAjRs3y9ecREREqoq3qxAREVGxlZycjDVrVsDVtRamTJkBgUCgNO7lyxf466//oV27jtDW1oa9fRVUrmyL4ODj8PEZBjU1NQBAQkICAMDAwEBhDAMDA4SGPpYd6+vrw9u7B2rWdIGBgSGePo3Ezp2/YdiwQfD334hq1RzylH+jRu5ybaVLl8GyZf7Q1dVTiP/ppz5yx9ra2pg+fXaBb5chIiJSNSxyEBERUbGlo6ODKlWq4c6d2zhxIki2QuO/sldstG7dTtbWpk07rFq1HNevX0XduvX/c4byYsnnRRSx2B5isb3s2NGxJurVa4C+fbth48a1WLlyLQAgMzMTUqlUFicUCmW3wujo6MLff/3/x0kQFfUau3b9hvHjR2L16o0oV6683PwzZsxD+fJZbe/fv8OJE8cwf/4saGpqomHDJjm+TkRERCUFb1chIiKiYktNTQ1LlqyAk5MzFi+ehxMnghRiMjMzERx8HKVLl0HFitZISkpCUlISPDzqQ01NTe6WFUNDQwBAYmKCwjiJiYlKV3h8ztTUDG5u7nj48J6sbfRoXzRq5C77s3DhnM/yF8Levirs7auiWjUHNG3aHL/+uhLv3sVhy5YNCuNXqlRJFl+nTj3Mnr0A5cqVx/LlS3J/sYiIiEoAruQgIiKiYk1bWxtLlqzApEljsGjRXAgEArRq1VbWf/PmNcTEvAUAtGrVWOH8q1cv4/379zA2NkalStYAgIiIMNSqJX8bSUREGBwcauSaj1QqwecrQSZNmoqUlBTZsaGh0RfPNzU1g6GhEcLCQnOdSygUomJFa5w//wfev38HY2OTXM8hIiJSZSxyEBERUbH3eaEje6VEdqHj+PGjUFNTw8KFyxSeuhIeHoaVK5fi1Knj6N69N0qVsoC9fVWcPh0Mb++esttKbt++iZiYt2jYULFI8rm4uFjcunVTbj+O8uUr5utaYmNjkJAQj4oVK+UaK5FIEBkZDk1NTaV7eBA
```

Comment:
I think plot will be better readable with legend on the right plt.legend(loc='upper right')

Reply:
@bluejurand corrected